plugins/mumble_plugin_utils.h

```diff
@@ -18,6 +18,7 @@
 // when constructing JSON documents via
 // string concatenation.
 static void escape(char *str) {
+    str[sizeof(str)-1] = 0; // NUL terminate queried C strings. We do this to ensure the strings from the game are NUL terminated. They should be already, but we can't take any chances.
```

Comment:
Instead, you could require callers of `escape` to pass in the size of the array as a second parameter -- and then use that for terminating.